### PR TITLE
CASSANDRA-17699 Updated broken link in development/documentation.html (updated)

### PR DESCRIPTION
**Pull request by msacks-dev**
this should be squashed and cleaner pull request with the proper link. please review and close PR 143 as it has incorrect code. this has been built and tested.

### CODE REVIEW

File: site-content/source/modules/ROOT/pages/development/documentation.adoc [hunk 1]
```diff
@@ -8,18 +8,17 @@
 The official Cassandra documentation lives in the project's git
 repository.
 We use a static site generator, http://www.antora.org/[Antora], to create pages hosted at
-link:/doc/latest/[cassandra.apache.org].
-
+https://cassandra.apache.org/docs/latest/[cassandra.apache.org].
```
Comment (michaelsembwever) [inline site-content/source/modules/ROOT/pages/development/documentation.adoc]: This wasn't broken before, and `link:` is to be favoured. Can you revert this change please.

File: site-content/source/modules/ROOT/pages/development/documentation.adoc [hunk 2]
```diff
@@ -8,18 +8,17 @@
...
 guide). -->
 
 Using a static site generator often requires the use of a markup language
 instead of visual editors (which some people would call good news).
-Antora processes http://www.asciidoc.org[Ascidoc], the markup language used to generate our documentation.
+Antora processes http://www.asciidoc.org[AsciiDoc], the markup language used to generate our documentation.
 Markup languages allow you to format text using certain syntax elements.
 Your document structure will also have to follow specific conventions.
-Feel free to take a look at http://cassandra.apache.org/docs[existing documents] to get a better idea how we structure our documents.
+Feel free to take a look at https://cassandra.apache.org/doc/latest/[existing documents] to get a better idea how we structure our documents.
```
Comment (michaelsembwever) [inline site-content/source/modules/ROOT/pages/development/documentation.adoc]: Please remove the `latest/` suffix, as agreed [here](https://github.com/apache/cassandra-website/pull/143#discussion_r904164956).
```suggestion
Feel free to take a look at https://cassandra.apache.org/doc/[existing documents] to get a better idea how we structure our documents.
```

File: site-content/source/modules/ROOT/pages/development/documentation.adoc (hunk 1 above)
Comment (msacks-dev) [reply site-content/source/modules/ROOT/pages/development/documentation.adoc]: Will do.

File: site-content/source/modules/ROOT/pages/development/documentation.adoc [hunk 3]
```diff
@@ -18,7 +18,7 @@ instead of visual editors (which some people would call good news).
 Antora processes http://www.asciidoc.org[AsciiDoc], the markup language used to generate our documentation.
 Markup languages allow you to format text using certain syntax elements.
 Your document structure will also have to follow specific conventions.
-Feel free to take a look at https://cassandra.apache.org/doc/latest/[existing documents] to get a better idea how we structure our documents.
+Feel free to take a look at link:/doc/latest/[existing documents] to get a better idea how we structure our documents.
```
Comment (michaelsembwever) [inline site-content/source/modules/ROOT/pages/development/documentation.adoc]: this changed the wrong link. we want this one to remain as `https://cassandra.apache.org/doc/` (to demonstrate the versions of docs we have available, i.e. "the structure")

it was this [one](https://github.com/apache/cassandra-website/pull/157/files#r939723811) that needed reverting

**Review by msacks-dev — COMMENTED** [review]
testing locally now.

File: site-content/source/modules/ROOT/pages/development/documentation.adoc (hunk 3 above)
Comment (michaelsembwever) [reply site-content/source/modules/ROOT/pages/development/documentation.adoc]: can you try the following please
```suggestion
Feel free to take a look at link:/doc/[existing documents] to get a better idea how we structure our documents.
```

File: site-content/source/modules/ROOT/pages/development/documentation.adoc [hunk 4]
```diff
@@ -9,17 +9,16 @@ The official Cassandra documentation lives in the project's git
 repository.
 We use a static site generator, http://www.antora.org/[Antora], to create pages hosted at
 link:/doc/latest/[cassandra.apache.org].
-
```
Comment (michaelsembwever) [inline site-content/source/modules/ROOT/pages/development/documentation.adoc:12]: We need this blank line restored. (i.e. please don't remove it)

Because removing it makes it look like…

Screenshot 2022-08-14 at 13 24 34

while originally it looks like (which is how we want it)

Screenshot 2022-08-14 at 13 26 09

**Comment by msacks-dev** [comment]
i added the whitespace and tested locally, should be good now?

**Review by michaelsembwever — APPROVED** [review]

**Comment by michaelsembwever** [comment]
Love it! Thanks for sticking at it @msacks-dev

**Comment by michaelsembwever** [comment]
merged manually with 11349c1a
